src/features/ShowMovie/ui/Projection.tsx

```diff
@@ -10,14 +10,18 @@ interface ProjectionProps {
     city: string;
     cinema: string;
     timeSlots: string[];
+    isVisible: boolean;
 }
 
-export const Projection: FC<ProjectionProps> = ({ city, cinema, timeSlots }) => {
+export const Projection: FC<ProjectionProps> = ({ city, cinema, timeSlots, isVisible }) => {
     const [isContainerHovered, setIsContainerHovered] = useState(false);
 
     return (
         <div
-            className="flex flex-col xl:flex-row xl:gap-60 gap-12 p-20 items-center ml-6 border-b-2"
+            className={clsx(
+                'flex flex-col xl:flex-row xl:gap-60 gap-12 p-20 items-center ml-6 border-b-2',
+                isVisible ? '' : 'hidden'
```

Comment:
you can use an object as second param for clsx 
```suggestion
                { 'hidden': isVisible }
```